README.md

```diff
@@ -29,20 +29,24 @@ This document explains how to install the CityEngine SDK and how to work with th
       examples/...
    ```
 
-3. Continue with building and running the `prt4cmd` example for either [Linux](examples/README_linux.md) or [Windows](examples/README_windows.md).
+3. Continue with building and running the `prt4cmd` example for either [Windows](examples/prt4cmd/README_windows.md) or [Linux](examples/prt4cmd/README_linux.md). This will automatically download the SDK archive for your platform.
 
 ## Table of Contents
 
-* [Downloads](https://github.com/Esri/cityengine-sdk/releases)
+* [Downloads](#downloads)
 * [Examples](#examples)
-* [API Documentation](https://esri.github.io/cityengine-sdk)
-* [Software Requirements](#general-software-requirements)
+* [Documentation](https://esri.github.io/cityengine-sdk)
 * [Related Software Products](#related-software-products)
+* [General Software Requirements](#general-software-requirements)
 * [Release History and Changelog](#release-history-and-changelog)
 * [CityEngine Resources](#cityengine-resources)
 * [Issues and Contributions](#issues-and-contributions)
 * [Licensing](#licensing)
 
+## Downloads
+
+Download the SDK archive, the open-source examples in this repository and the example data from the Assets section on the [Releases](https://github.com/Esri/cityengine-sdk/releases/latest) page.
```

Comment:
I'd rather go with "You can download ..." than "Download ..."
After all the user does not need to download the SDK zips himself anymore.